src/sym.py

```diff
@@ -0,0 +1,78 @@
+import math
+
+'''
+Class Sym which handles symmbolic characters 
+'''
+class sym:
```

Comment:
Class name should be starting with a capital letter as per the [generic naming convention in Python](https://peps.python.org/pep-0008/#class-names)